build/images/Dockerfile.build.windows

```diff
@@ -48,11 +22,11 @@ LABEL description="A Docker image to deploy the Antrea CNI."
 USER ContainerAdministrator
 
 RUN mkdir -Force C:\k\antrea\bin
-COPY --from=cni-binaries-windows  /opt/cni/bin /k/antrea/cni
+COPY --from=antrea-build-windows  /opt/cni/bin /k/antrea/cni
 COPY --from=antrea-build-windows  /antrea/build/images/scripts/Install-WindowsCNI.ps1 /k/antrea/
 COPY --from=antrea-build-windows  /antrea/bin/antrea-agent.exe /k/antrea/bin/
 COPY --from=antrea-build-windows  /antrea/bin/antctl-windows.exe /k/antrea/bin/antctl.exe
```

Comment:
We don't have antctl-windows.exe now, it is called antctl.exe now.

Reply:
Updated, thanks.